src/app/submit-item/components/submit/submit.component.scss

```diff
@@ -32,18 +32,25 @@ textarea {
 form.form-centered {
   display: flex;
   flex-direction: column;
-  align-items: center;
+  align-items: left;
 }
```

Comment:
form-centered aka. form-left-aligned :D

Reply:
Damn - I knew I forgot something. Thanks for pointing this out :D